### ISSUE TITLE
show black log and increase by the time

### ISSUE THREAD

**Issue by larrycai** [ArchCI/archci#22]
In ArchCI build screen, the place for log shows nothing (black), even it keeps growing by the time.

see snapshot

![capture](https://cloud.githubusercontent.com/assets/106192/9020177/415a2b94-3830-11e5-9f60-dfafafdff883.PNG)

**Comment by tobegit3hub** [comment]
Yep, It's the known issue which is marked `TODO`. Thanks for reporting @larrycai
